apps/aecontract/src/aect_contracts.erl

```diff
@@ -96,6 +100,11 @@
 id(C) ->
   pubkey(C).
 
+-spec store_id(contract()) -> id().
+store_id(C) ->
+    CId = pubkey(C),
+    << CId/binary, ?STORE_PREFIX>>.
```

Comment:
The `?STORE_PREFIX` placed as a suffix of the contract ID. I see why, but is should have a comment explaining it.